omaha/common/brave_stats_updater.h

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef OMAHA_COMMON_BRAVE_STATS_UPDATER_H__
+#define OMAHA_COMMON_BRAVE_STATS_UPDATER_H__
+
+namespace omaha {
+
+HRESULT BraveSendStatsPing(const CString& event, const CString& app_guid,
```

Comment:
I think caller doesn't need to pass app guid to this function because this function can get app guid by itself.

Reply:
Right, good idea. Fixed.